deltacat/benchmarking/README.md

```diff
@@ -0,0 +1,18 @@
+# Benchmarking
+
+## Setup
+
+1. Install DeltaCat's `requirements.txt` and `dev-requirements.txt`
```

Comment:
What do you think about introducing a new `benchmark-requirements.txt` that includes `dev-requirements.txt` and daft, then just changing this line to tell the reader to just install `benchmark-requirements.txt`?